### DISCUSSION TITLE
No Student Access?

### DISCUSSION THREAD

**Discussion by maviddoerdijk** [community/community#118524] [GitHub Education]
### Select Topic Area

Bug

### Body

I used Copilot for months. It was great.

A few days ago, my github student pack didn't work anymore, so I had to request it again. It was permitted, but I still don't have access to copilot now.

According to the sudent pack:
```
To enable Copilot, go to your account settings and under Code, planning, and automation, select Copilot. You'll see "You're eligible to use copilot for free" and a button to claim the offer.
```

But in my account it doesn't show any button to claim the offer. 

Anyone else having this problem?

**Comment by AishaMushtaq0** [comment]
If you're facing issues accessing Copilot despite being eligible through your GitHub student pack:
1. Double-check eligibility criteria.
2. Contact GitHub support for assistance.
3. Seek help on community forums.
4. Be patient as technical issues are resolved.
source: [gbwhatsapp](https://wasuppgb.com/)
![images](https://github.com/community/community/assets/164985976/d352dbc2-c09b-41ed-affe-b870a9b08776)

**Comment by ScripterDude** [comment]
im experiecing the same issue

**Comment by RajShah02** [comment]
Hello guys,
Did you find any solution about the student pack. I am facing the same issue with my pack. I was thinking to share it on [GB WhatsApp](https://gbwa.org.in/) community, but thought before that i should drop a message here.

**Comment by VictorMoni** [comment]
Hi @maviddoerdijk 

You should contact [GitHub Education Support](https://support.github.com/contact/education) and select `Other` as the category. They will be able to fix this for you.
